web/src/components/columnGrid.js

```diff
@@ -29,11 +29,16 @@ const Grid = styled.div`
     ${tw`grid grid-cols-1 lg:grid-cols-3 gap-24 lg:px-16`}
 `;
 
-export default function ColumnGrid({ content }) {
-    const { heading, indexIndustries } = content;
+const Description = styled.p`
+    ${tw`text-xl text-black text-center mb-12`}
+`;
+
+export default function ColumnGrid({ content, hasDescriptionText }) {
+    const { description, heading, indexIndustries } = content;
     return (
         <Section>
             <Heading>{heading}</Heading>
+            {hasDescriptionText && <Description>{description}</Description>}
```

Comment:
Can we just use `{description && ...}` here? If you pass in an object that doesn't have a `description` key what does `description` evaluate to?

Reply:
@asasmith isnt that how I had it before? without the prop?

Reply:
You weren't using a render prop to conditionally display the content

Reply:
It looks like this content isn't making it to the ui now either.
<img width="1404" alt="Screen Shot 2021-03-09 at 4 10 56 PM" src="https://user-images.githubusercontent.com/11489171/110538449-0e94b600-80f2-11eb-996b-e13322aaaa2d.png">

Reply:
In the `content` object you have a `body` key, not a `description` key.

Reply:
You weren't using a render prop before so the dom element was present even if there wasn't any content.

Reply:
@asasmith thats not for that component, this is being queried from sanity. Once I can get into sanity I can add the description, query it and it will render.

Reply:
@asasmith i just pushed up an example for you but here i queried the same title, just below with the styled description
<img width="1020" alt="Screen Shot 2021-03-09 at 5 06 31 PM" src="https://user-images.githubusercontent.com/61708936/110544528-de511580-80f9-11eb-96a0-c75ef46efc76.png">

Reply:
Oh, my mistake. I thought you were passing the `content` object to this component.

Reply:
Still, `{description && ...}` should work. We don't need the extra prop here.